cmd/gb-vendor/fetch.go

```diff
@@ -70,4 +99,5 @@ var FetchCmd = &cmd.Command{
 		}
 		return wc.Destroy()
```

Comment:
@davecheney I've noticed that you are returning wc.Destroy() here. Wouldn't it be more intuitive if you return vendor.WriteManifest(...)? Since the wc.Destroy() is not the main objective of the command. Atleast I've used that approach in the update and delete command.